fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/TransactionsIT.java

```diff
@@ -224,7 +225,7 @@ public void testCreateDoStuffAndRollbackTransaction() throws IOException {
     }
 
     @Test
-    public void testRollbackShouldNotLeaveDbInPartiallyUpdatedState() throws IOException {
+    public void conflictingContainmentOverwritesObjectModifiedInTransactionRdf() throws IOException {
```

Comment:
Yes these are testing something different. I'm not sure how we cause a rollback now. The previous tests were all adding a conflicting record then commiting the transaction. But now instead of failing out we just update the record instead.

Reply:
Yes... I am struggling to think of a good way to make one of those db commits fail. I suspect that the most likely real world scenario is a failed db connection, which is hard to simulate here without a mock. What if you drop one of the transaction tables before committing? The table should be recreated automatically before the next test runs.